htdocs/takepos/invoice.php

```diff
@@ -207,7 +207,8 @@ function fail($message)
 			dol_htmloutput_errors($langs->trans("InvoiceIsAlreadyValidated", "TakePos"), null, 1);
 		}
 	} elseif (count($invoice->lines) == 0) {
-		dol_syslog("Sale without lines");
+		$error++;
+		dol_syslog('Sale without lines', LOG_ERR);
```

Comment:
The LOG_ERR like LOG_WARNING must be used to report to developer of sysadmin there is an error into setup or a bug somewhere.
A common use of Dolibarr should never trigger such error.
My understanding is that this case can occurs just with a bad use of end user. In such a case, we should report an error on screen for user telling him something is missing be we should not alert the system supervisor. Don't you think so ?

Reply:
I understand for log level in syslog.
But we have to prevent from adding a payment on invoice without lines.
So I should only remove 'LOG_ERR' in syslog.
Is it right ?

Reply:
Yes, only the LOG_ERR to remove